torch/_torch_docs.py

```diff
@@ -2690,6 +2718,34 @@ def merge_dicts(*dicts):
     tensor([1, 0, 0], dtype=torch.int16)
 """.format(**common_args))
 
+add_docstr(torch.logical_or,
+           r"""
+logical_or(input, other, out=None) -> Tensor
+
+Computes the element-wise logical OR of the given input tensors. Zeros are treated as ``False`` and nonzeros are
+treated as ``True``.
+
+Args:
+    {input}
+    other (Tensor): the tensor to compute OR with
+    {out}
+
+Example::
+
+    >>> torch.logical_or(torch.tensor([True, False, True]), torch.tensor([True, False, False]))
+    tensor([ True, False,  True])
+    >>> a = torch.tensor([0, 1, 10, 0], dtype=torch.int8)
+    >>> b = torch.tensor([4, 0, 1, 0], dtype=torch.int8)
+    >>> torch.logical_or(a, b)
+    tensor([ True,  True,  True, False])
```

Comment:
let's fix the extra spaces too so this actually looks like a direct paste

Reply:
I think this is pretty print as it is what was printed from my Python shell.

Reply:
ah I see, sorry, yeah I get the same when printing a bool tensor.